### DISCUSSION TITLE
[Feature Proposal]: Add "max-height" utilities similar to existing "max-width" options

### DISCUSSION THREAD

**Discussion by soullivaneuh** [tailwindlabs/tailwindcss#1374] [Ideas]
Basically, the same thing than that: https://tailwindcss.com/docs/max-width/#app

But for height.

I might add it on a tailwindcss config file, but I'm working on an open-source svelte library based on Tailwind, and I would keep the strict default config to be flexible.

This will be useful to set the max height of a dropdown. Currently, I have to do this:

```css
.list {
  max-height: 16rem;
}
```

**Comment by hunzaboy** [comment]
You can use the height on the parent and then use max-height in the child. 

`h-64 = 16rem;`
e.g 

```
<div class="h-64 list">
  <div class="max-h-full ...">
    max-h-full
  </div>
</div
```

**Comment by kyleoliveiro** [comment]
> You can use the height on the parent and then use max-height in the child.
> 
> `h-64 = 16rem;`
> e.g
> 
> ```
> <div class="h-64 list">
>   <div class="max-h-full ...">
>     max-h-full
>   </div>
> </div
> ```

In that case the parent will occupy more space than intended if the child is shorter than 16rem.
